components/automate-cli/cmd/chef-automate/applications.go

```diff
@@ -168,7 +171,14 @@ func newApplicationsRemoveSvcsCmd() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "remove-svcs",
 		Short: "Remove services from the applications database",
-		RunE:  runApplicationsRemoveSvcsCmd,
+		Long: `
+Remove services from the applications database.
+
+You must decommission the services prior to running this command. Services
```

Comment:
```suggestion
You must fully decommission services by retiring physical hardware, terminating the VM or container, or by using 'hab svc unload' before using the 'remove-svcs' command. Services that are not fully decommissioned will send a health-check at the appointed time and Automate will re-add them to the services database.
```

Reply:
It is wordy, but tells a new user what needs done and the consequences of doing it badly.